src/character.cpp

```diff
@@ -3957,6 +3957,16 @@ bool Character::is_wearing( const itype_id &it ) const
     return false;
 }
 
+bool Character::is_wearing( const item &it ) const
```

Comment:
There's `Character::is_worn` in `character.h` with exactly the same function.

Reply:
thank you, testing it right now ... i ll swap the call out to is_worn and remove the new method